libs/components/src/lib/radio-group/radio-group.spec.ts

```diff
@@ -63,6 +63,16 @@ describe('vwc-radio-group', () => {
 			expect(radios[1].getAttribute('tabindex')).toBe('-1');
 			expect(radios[2].getAttribute('tabindex')).toBe('-1');
 		});
+
+		it('should use fieldset with proper legend description', async () => {
+			element.label = 'test label';
+			await elementUpdated(element);
+			const fieldset = element.shadowRoot?.querySelector('fieldset');
+			const legend = fieldset?.querySelector('legend');
+
+			expect(fieldset).not.toBeFalsy();
+			expect(legend).not.toBeFalsy();
```

Comment:
```suggestion
			expect(legend).toBeTruthy();
```

Just a suggestion - in other tests we're using toBeTruthy to check whether an element exists in the DOM